andromedocs-theme/src/components/templates/docs.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { graphql } from "gatsby";
+import { MDXProvider } from "@mdx-js/react";
+import { MDXRenderer } from "gatsby-plugin-mdx";
+
+import Layout from "../layout/layout";
+
+const Docs = ({ data: { mdx } }) => (
+  <>
+    <Layout title={mdx.frontmatter.title}>
+      <MDXProvider>
+        <MDXRenderer>{mdx.body}</MDXRenderer>
+      </MDXProvider>
+    </Layout>
+  </>
```

Comment:
```suggestion
  <Layout title={mdx.frontmatter.title}>
    <MDXProvider>
      <MDXRenderer>{mdx.body}</MDXRenderer>
    </MDXProvider>
  </Layout>
```
You can remove this Fragment, since you're not grouping the Layout with anything :sparkles:

Reply:
Thanks, I didn't even realize it, but I'll fix it :satisfied:

Reply:
Resolves 34d9daa